frame/election-provider-multi-phase/src/lib.rs

```diff
@@ -1571,6 +1586,11 @@ impl<T: Config> Pallet<T> {
 		<Round<T>>::mutate(|r| *r += 1);
 
 		// Phase is off now.
+		Self::deposit_event(Event::PhaseTransition {
```

Comment:
Similarly I would make a helper function here that sets the phase and emits the right event.